components/Downloads/SecondaryDownloadMatrix.tsx

```diff
@@ -63,7 +63,9 @@ const SecondaryDownloadMatrix: FC<SecondaryDownloadMatrixProps> = ({
         </tbody>
       </table>
 
-      <DownloadList node={node} />
+      <WithNodeRelease status={status}>
```

Comment:
There is another way of writing this:

```js
<WithNodeRelease status={status}>
  {DownloadList}
</WithNodeRelease>
```

```js
type DownloadListProps {
  release: NodeRelease;
}

const DownloadList: FC<DownloadListProps> = (release: { versionWithPrefix }) => {
```

I choose this way because we don't need to introduce an additional Props type.

Reply:
Yup, way cleaner and simpler :)